charms/trusty/clearwater-aio-proxy/lib/reconfigure-aio

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Reconfigures an all-in-one image to use a new home domain and number range.
+# Usage: reconfigure-aio <home-domain> [<base-number> <number-count>]
+
+# Get command-line arguments.
+home_domain=$1
+base_number=$2
+number_count=$3
+
+if [ -z "$home_domain" ] ; then
+  echo "Usage: reconfigure-aio <home-domain> [<base-number> <number-count>]"
+fi
+
+# Remove all old numbers from the database, unless they're currently assigned.
+old_home_domain=$(. /etc/clearwater/config ; echo $home_domain)
+echo "DELETE FROM ellis.numbers WHERE number LIKE '%@$old_home_domain' AND owner_id IS NULL ;" | mysql
+
+# Update /etc/clearwater/shared_config.
+function escape { echo $1 | sed -e 's/\//\\\//g' ; }
+sed -e 's/^home_domain=.*$/home_domain='$(escape $home_domain)'/g' \
```

Comment:
No, but if you were to use it, the script would break in a bizarre way - better to protect against this.  (Better still to police against it, but we don't have a great way of doing that here.)